src/service/store/state/state-receipt.ts

```diff
@@ -29,6 +38,7 @@ export class StateReceipt {
 
   async add(receiptId: string, date: Date = new Date()): Promise<void> {
     this.state.set(receiptId, date);
+    this.status()
```

Comment:
what's the purpose of this?

Reply:
set the new status after adding a new receipt
added it to the reset() method as well